app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationActions.jsx

```diff
@@ -63,6 +63,16 @@ function V3csvExport(selected, registrations, competition) {
   URL.revokeObjectURL(url);
 }
 
+function escapeCsv(value) {
+  if (value == null) return '';
+  const str = String(value);
+  const escaped = str.replace(/"/g, '""');
```

Comment:
Why are you replacing one quote with two quotes? Is that really the way to escape quotes in CSV?

Reply:
If yes, please provide an additional code comment explaining this behavior.